lib/basedriver/image-helpers.js

```diff
@@ -1,32 +1,69 @@
-import {PNG} from 'pngjs';
+import { Buffer } from 'buffer';
+import { PNG } from 'pngjs';
+import B from 'bluebird';
 
-function cropBase64Image (base64EncodedImage, rect) {
-  let pngImage = PNG.sync.read(Buffer.from(base64EncodedImage, 'base64'));
+async function createImageFromBase64 (base64EncodedImage) {
+  const buffer = Buffer.from(base64EncodedImage, 'base64');
```

Comment:
this method is only available since node 8

Reply:
Changed to default buffer constructor